src/miner.erl

```diff
@@ -638,7 +633,7 @@ block_time(Block, Metadata) ->
 -spec select_transactions(
     blockchain:blockchain(),
     blockchain_txn:txns(),
-    blockchain_block:block(),
+    {non_neg_integer(), non_neg_integer()},
```

Comment:
```suggestion
    ElectionInfo :: {non_neg_integer(), non_neg_integer()},
```

Reply:
not needed per se

Reply:
Although they're already named in the func def, so never mind.